src/utils/sources-tree/tests/getUrl.spec.js

```diff
@@ -85,52 +82,4 @@ describe("getUrl", () => {
     );
     expect(urlObject.group).toBe("webpack://");
   });
-
-  describe("memoized", () => {
-    beforeEach(() => {
-      spy = jest.spyOn(Url, "parse");
-    });
-
-    afterEach(() => {
-      spy.mockReset();
-      spy.mockRestore();
-    });
-
-    it("parses a url once", () => {
-      const source = createMockSource({
-        url: "http://example.com/foo/bar/baz.js"
-      });
-
-      getURL(source);
-      const url = getURL(source);
-      expect(spy).toHaveBeenCalledTimes(1);
-
-      expect(url).toEqual({
-        filename: "baz.js",
-        group: "example.com",
-        path: "/foo/bar/baz.js"
-      });
-    });
-
-    it("parses a url once per source", () => {
-      const source = createMockSource({
-        url: "http://example.com/foo/bar/baz.js"
-      });
-      const source2 = createMockSource({
-        id: "server1.conn13.child1/40",
-        url: "http://example.com/foo/bar/baz.js"
-      });
-
-      spy = jest.spyOn(Url, "parse");
-      getURL(source);
-      const url = getURL(source2);
-      expect(spy).toHaveBeenCalledTimes(2);
-
-      expect(url).toEqual({
-        filename: "baz.js",
-        group: "example.com",
-        path: "/foo/bar/baz.js"
-      });
-    });
-  });
```

Comment:
Hmm, it'd still nice to make sure we don't parse too often.

perhaps we need to spy on `./utils/url` parse. Jest has pretty good module mocking

Reply:
Normally when I do this kind of mocking I use `jest.mock('module')` and if I need to specifically mock one of its exports I can do `const mockSpy = jest.fn(); jest.mock('module', () => ({ theExport: mockSpy }));`